### ISSUE TITLE
apt upgrade / apt update / apt install not recognized

### ISSUE THREAD

**Issue by mschiessl** [hadolint/hadolint#318]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
apt [upgradelupdate|install|...] is exactly treated like apt-get [upgrade|update|install|...] 

### Actual behavior
using `apt-get update && apt-get upgrade`is throwing the expected errors, by using `apt update && apt upgrade` the check is bypassed

### Steps to reproduce the behavior

Create 2 dockerfiles:
dectected correctly:
```
FROM debian:9.9
RUN apt-get update && apt-get upgrade && apt-get install curl 
```
not detected
```
FROM debian:9.9
RUN apt update && apt upgrade && apt install curl 
```

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
v1.16.3
```

Sorry i am not really into haskell, so i am not able to send a PR

**Comment by stephengroat-dd** [comment]
I ran into this same issue as well and was thinking the recommendation should first be to use `apt-*` binaries over `apt`.

According to the [`apt` manpage](https://manpages.debian.org/stretch/apt/apt.8.en.html#SCRIPT_USAGE_AND_DIFFERENCES_FROM_OTHER_APT_TOOLS):

>The apt(8) commandline is designed as an end-user tool and it may change behavior between versions. While it tries not to break backward compatibility this is not guaranteed either if a change seems beneficial for interactive use.
>All features of apt(8) are available in dedicated APT tools like apt-get(8) and apt-cache(8) as well. apt(8) just changes the default value of some options (see apt.conf(5) and specifically the Binary scope). So you should prefer using these commands (potentially with some additional options enabled) in your scripts as they keep backward compatibility as much as possible.

From my reading of this, especially the mentions of "end-user tool", changing behavior, etc., it seems like a rule recommending against `apt` and to use `apt-get` and `apt-cache` specific binaries would be better

**Comment by zemanlx** [comment]
@stephengroat-dd sounds good. Accepting just `apt` would be easier, but not using `apt` instead of `apt-get` sound like a good rule. What do you think @lorenzo?

**Comment by lorenzo** [comment]
Sounds like discouraging apt is the way to go
